src/generalizedlinearmixedmodel.jl

```diff
@@ -567,6 +578,16 @@ function Base.show(io::IO, m::GeneralizedLinearMixedModel)
     show(io, coeftable(m))
 end
 
+function σs(m::MixedModel)
+    σ = dispersion(m)
+    NamedTuple{fnames(m)}(((σs(t, σ) for t in m.reterms)...,))
+end
+
+function σρs(m::MixedModel)
+    σ = dispersion(m)
+    NamedTuple{fnames(m)}(((σρs(t, σ) for t in m.reterms)...,))
+end
+
```

Comment:
should we have a separate `mixedmodel.jl` file for methods defined for the superclass?

Reply:
Yes.

Reply:
The trap I always fall into here is using both the upper-case and lower-case names `MixedModels.jl` and `mixedmodels.jl` then finding out that certain operating systems still use case-insensitive file names.  Very important for those who still want to do their computing on punched cards.

Reply:
This also occurred to me after I reviewed ... but then we have the distinction between `MixedModels.jl` for the package and `mixedmodel.jl` for the class. While I typically don't like have an important identifier differ only singular/plural, I can't think of a better name for the file with the superclass methods.